src/suites/cts/validation/error_scope.spec.ts

```diff
@@ -32,14 +32,20 @@ class F extends Fixture {
     this.device.getQueue().submit([]);
   }
 
-  async expectUncapturedError(fn: Function): Promise<void> {
+  async expectUncapturedError(fn: Function, callback: Function): Promise<void> {
     // TODO: Make arbitrary timeout value a test runner variable
     const TIMEOUT_IN_MS = 1000;
 
     return this.asyncExpectation(async () => {
       try {
         const promise = new Promise(resolve => {
-          this.device.addEventListener('uncapturederror', resolve, { once: true });
+          const eventListener = ((event: GPUUncapturedErrorEvent) => {
+            this.debug(`Got uncaptured error event with ${event.error}`);
+            callback(event);
```

Comment:
Instead of passing an extra callback, could we just resolve with the value of the event? In fact, despite the `: Promise<void>` at the top, that's probably what was already happening, because we passed `resolve` directly into addEventListener.